lib/codegen/src/context.rs

```diff
@@ -177,31 +177,43 @@ impl Context {
     ///
     /// Also check that the dominator tree and control flow graph are consistent with the function.
     pub fn verify<'a, FOI: Into<FlagsOrIsa<'a>>>(&self, fisa: FOI) -> VerifierResult<()> {
-        verify_context(&self.func, &self.cfg, &self.domtree, fisa)
+        let mut errors = VerifierErrors::default();
+        let _ = verify_context(&self.func, &self.cfg, &self.domtree, fisa, &mut errors);
+
+        if errors.is_empty() {
+            Ok(())
+        } else {
+            Err(errors)
+        }
```

Comment:
Can this use `verify!`?

Reply:
`context` is imported before `verifier` therefore the `verify!` macro doesn't exist yet when this is parsed, so either we change the order at which the modules are loaded or we keep this.